.github/workflows/verify_library_generation.yaml

```diff
@@ -36,17 +51,10 @@ jobs:
 
         # launch docker
         sudo systemctl start docker
-    - name: install docker (macos)
-      if: matrix.os == 'macos-12'
-      run: |
-        brew update --preinstall
-        brew install docker docker-compose qemu 
-        brew upgrade qemu
-        colima start
-        docker run --user $(id -u):$(id -g) --rm hello-world
     - name: Run integration tests
+      if: matrix.os == 'ubuntu-22.04' || matrix.post_processing == 'false'
```

Comment:
Can you add a reason why you don't run the integration tests in this condition?

Reply:
done